docs/source-fabric/index.rst

```diff
@@ -1,57 +1,77 @@
 .. include:: links.rst
 
-####################
-Welcome to ⚡ Fabric
-####################
+#############################
+Welcome to ⚡ Lighting Fabric
+#############################
+
+.. twocolumns::
+   :left:
+      .. image:: https://pl-bolts-doc-images.s3.us-east-2.amazonaws.com/mov.gif
+         :alt: Animation showing how to convert standard training code to Lightning
+   :right:
+      Lightning Fabric is designed to enable the flexibility and control needed to scale billion-trillion parameter models.
```

Comment:
This is not what Fabric was designed to do at its core.

When training billion parameter models became a need for users, Lightning Trainer has been updated with new capabilities to do that long before Lightning Lite and Fabric has cristalized itself. 

Fabric solves a different problem, which is giving users full control over the training loop. Of course, it can also do the billion parameter model training but that's not the core design philosophy.

Reply:
could you pls send suggestion for better description?